CWE/inventions/culture_inventions.txt

```diff
@@ -925,26 +977,26 @@ terrorism = {
 	effect = {
 		enable_crime = terrorist_cells 
 		enable_crime = police_brutality
-        rebel_org_gain = {
-            faction = islamist_rebels
-            value = 0.75
-        }		
+		rebel_org_gain = {
+			faction = islamist_rebels
+			value = 0.25
+		}		
 	}
 }
 identity_politics = {
 	limit = { political_polarization = 1 }
+	news = no
 	chance = {
 		base = 5
 	}
 	effect = { 
-        rebel_org_gain = {
-            faction = populist_rebels
-            value = 0.75
-        }
```

Comment:
rebel org gain should be retained but changed to affect sectarian_rebels

Reply:
Fixed it.